pkgs/development/compilers/rust/cargo.nix

```diff
@@ -38,6 +38,9 @@ rustPlatform.buildRustPackage {
       --set SSL_CERT_FILE "${cacert}/etc/ssl/certs/ca-bundle.crt"
 
     installManPage src/tools/cargo/src/etc/man/*
+
+    installShellCompletion --bash --name cargo \
+      src/tools/cargo/src/etc/cargo.bashcomp.sh
```

Comment:
would you mind adding the zsh completion?
```
installShellCompletion --zsh src/tools/cargo/src/etc/_cargo
```

Reply:
Added in the other PR.